proposals/csharp-9.0/init.md

```diff
@@ -242,6 +242,27 @@ Restrictions of this feature:
 - All overrides of a property must have `init` if the base had `init`. This rule
 also applies to interface implementation.
 
+### Readonly structs
+
+`init` accessors (both auto-implemented accessors and manually-implemented
```

Comment:
As for manually-implemented accessors, please see https://github.com/dotnet/csharplang/issues/3884#issuecomment-714987528